eng/common/scripts/Create-APIReview.ps1

```diff
@@ -130,6 +130,13 @@ if ($packages)
             }
             elseif ($version.IsPrerelease)
             {
+                # Check if package name is approved. Preview version cannot be released without pacakge name approval
+                if ($respCode -eq '202' -and $pkgInfo.ReleaseStatus -ne "Unreleased")
```

Comment:
Below I see you added `!$pkgInfo.ReleaseStatus` which leads me to believe that there are cases where ReleaseStatus is null so might be worth adding that here too.

Reply:
You are right. It's possible if there is no log for current version in change log.